docs/data-sources/masking_policies.md

```diff
@@ -2,42 +2,177 @@
 page_title: "snowflake_masking_policies Data Source - terraform-provider-snowflake"
 subcategory: ""
 description: |-
-  
+  Datasource used to get details of filtered masking policies. Filtering is aligned with the current possibilities for SHOW MASKING POLICIES https://docs.snowflake.com/en/sql-reference/sql/show-masking-policies query. The results of SHOW and DESCRIBE are encapsulated in one output collection masking_policies.
```

Comment:
I think you forgot about new templates indicating this is v1 candidate. Please, also check other resources you worked on recently, because It may be the same case with those If I remember correctly.

Reply:
Right, it was also missing in views - added.